modules/minimum-ibm-power-system-requirements.adoc

```diff
@@ -13,7 +13,7 @@ You can install {product-title} version {product-version} on the following IBM h
 
 [NOTE]
 ====
-Support for {op-system} functionality for all IBM POWER8 models, IBM POWER9 AC922, IBM POWER9 IC922, and IBM POWER9 LC922 is deprecated. These hardware models remain fully supported in {product-title} 4.12. However, Red Hat recommends that you use later hardware models.
+Support for {op-system} functionality for all IBM Power8 models, IBM Power AC922, IBM Power IC922, and IBM Power LC922 is deprecated. These hardware models are no longer supported in {product-title} {product-version}. Red Hat recommends that you use later hardware models.
```

Comment:
@alishaIBM doesn't this note kind of contradict itself now? The first sentence says that support is deprecated, and then the second says that they are not supported at all.

Should the first two sentences be combined to say that these are no longer supported?

Reply:
@bergerhoffer I have updated the above sentence. Request you to check. Thanks.